tests/Integration/Console/Scheduling/SubMinuteSchedulingTest.php

```diff
@@ -131,11 +131,11 @@ public function test_sub_minute_events_stop_for_the_rest_of_the_minute_once_main
         Sleep::whenFakingSleep(function ($duration) use ($startedAt) {
             Carbon::setTestNow(now()->add($duration));
 
-            if (now()->diffInSeconds($startedAt) >= 30 && ! $this->app->isDownForMaintenance()) {
+            if ($startedAt->diffInSeconds() >= 30 && ! $this->app->isDownForMaintenance()) {
                 $this->artisan('down');
             }
 
-            if (now()->diffInSeconds($startedAt) >= 40 && $this->app->isDownForMaintenance()) {
+            if ($startedAt->diffInSeconds() >= 40 && $this->app->isDownForMaintenance()) {
                 $this->artisan('up');
             }
```

Comment:
Changes to this file can be applied to Carbon 2 and Laravel 10, right?

Reply:
Yes. 👍

Reply:
Extracted #49785